docs/design-language.md

```diff
@@ -5,44 +5,62 @@ title: Design Language
 
-<div class="buttons-box"><ButtonStyles />
+Depending on the purpose of the button you can use combinations of a High Emphasis and Medium Emphasis button, two Medium Emphasis buttons, or a Medium and Low Emphasis buttons.
 
-<DisabledStates />
+<div class="buttons-box"><ButtonPairs />
 </div>
 
-### Button Colors
+#### Developer notes:
+
+-   Low Emphasis buttons consist of MUI Text Variant buttons using the 'Primary' colour
+-   Medium Emphasis buttons consist of MUI Outlined Variant buttons using the 'Primary' colour
+-   High Emphasis buttons consist of MUI Contained Variant buttons using the 'Primary' colour
 
-Adding colour to your buttons can help direct the eye to a 'primary action' or a button that requires more attention than other buttons surronding it.
+Visit **[the Developer Center](https://developers.vertigisstudio.com/docs/mobile/tutorial-configure-button-click/)** for more details for implementing buttons
 
-<div class="buttons-box"><ButtonColors /></div>
+### Order and Alignment
 
-### Button Sizes
+The primary action (eg: Save, Apply, etc.) should be the first button in the hierarchy of buttons. In the context of a form, it should be the first thing a keyboard user will tab too after filling in all of the form inputs. The 'Cancel' button is the last button that should be displayed.
 
-Various sizes are available to ensure that regardless of what your button is used for, it will fit within the visual heirarchy.
+VS Web generally uses right alignment for most basic button implementations. Workflow forms uses left alignment for buttons by default.
 
-<div class="buttons-box"><ButtonSizes /></div>
+### Button Labels
 
-### Icon Button
+Always attempt to use concise and descriptive labels with three or fewer words for buttons. Button labels must use 'Title Case' where each main word has the first letter capitalized.
+
+<div class="button-examples box-cols">
+    <fieldset class="box-item example eg-do">
+        <legend>Do</legend>
+        <ButtonLabelsDo />
+    </fieldset>
+    <fieldset class="box-item example eg-dont">
+        <legend>Don't</legend>
+        <ButtonLabelsDont />
+    </fieldset>
+</div>
 
-If space is an issue, or there is no need for a description of the button to be added, there are iconographic buttons available.
+### Sizing and Spacing
 
-<div class="buttons-box"><IconButtons /></div>
+Buttons will adjust their width to the length of their content. Spacing (padding and margin) should not be manually set as they are pre-set in the Design System so that buttons can participate in the system wide density settings. Please refer to **[the Developer Center](https://developers.vertigisstudio.com/docs/web/configuration-layout-reference#presentation-attributes)** for more information on spacing.
```

Comment:
Padding should be left alone, but margin actually is something that is set by component that is rendering the buttons. 

The general rule is that a parent component controls the positioning, flow, size (sometimes) and spacing around its child components, but should never change the look or feel of the child components themselves (i.e. no CSS that affects anything within a component's borders) beyond whatever properties are provided for this purpose, such as emphasis.

Reply:
In the specific case of buttons, you should never set their size directly with CSS. Instead use small, medium, and large via the size prop. Same in most other cases like icons and such. The only time you would specify sizing in CSS might be say a min height on a tabs component's content area (things that generally stretch in size).

Reply:
Thanks for the clarification on the padding, margin, and sizing. I've updated the text to cover this better.

Reply:
Thanks, looks good.